fuzzers/ensemble/builder.Dockerfile

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google LLC
```

Comment:
Can you please rename this ensemble_afl? Renaming fuzzers will be bad so I think it's better to be descriptive now in case we want to add a regular ensemble that includes other fuzzers.

Reply:
Done. Sounds good!